pkg/framework/spi/kms/api.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package kms
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable KMS service
+type Creator func(prov Provider) (KMS, error)
```

Comment:
usage is not clear.

Reply:
the intention is this creator function will be used by the framework provider to add a KMS.. it will be used when plugging in these interfaces into the framework at agent start-up